parcels/tools/_helpers.py

```diff
@@ -49,5 +49,5 @@ def deprecated_made_private(func: Callable) -> Callable:
     return deprecated(
         "It has moved to the internal API as it is not expected to be directly used by "
         "the end-user. If you feel that you use this code directly in your scripts, please "
-        "comment on our tracking issue at <>."  # TODO: Add tracking issue
+        "comment on our tracking issue at <>.",  # TODO: Add tracking issue
```

Comment:
Is there no tracking issue yet, that we can link to here?

Reply:
Just updated :) . I'll make sure to completely clean up that tracking issue before release